background.js

```diff
@@ -1,11 +1,26 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
+"use strict";
 
-'use strict';
+// Kept this code for reference
+chrome.runtime.onInstalled.addListener(function () {
```

Comment:
JSYK, this uses the storage API, which requires additional permissions. In general it's preferable to minimize the amount of permissions that we request from Google. So I would probably remove this unless we need it.